android/app/src/main/res/drawable/newgradient.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If we want to use multiple gradients throughout the project, we should name them more semantically such as `gray_gradient.xml`, `dark_gray_gradient.xml`